### PR TITLE
Update OSX.gitignore

### PR DESCRIPTION
**Pull request by fdizile**

### CODE REVIEW

**Comment by shiftkey** [comment]
It looks like the line ending have been clobbered - making it unclear what the actual change is.

Could you see if you can restore those?

File: Global/OSX.gitignore [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.temp
```
Comment (shiftkey) [inline Global/OSX.gitignore]: Do you know what tool might be responsible for generating this `.temp` file?

Comment (DigiCoon) [inline Global/OSX.gitignore]: It's generated by any Adobe, Microsoft, or other applications of the sort
that have AppData.

On Sunday, July 5, 2015, Brendan Forster <notifications@github.com> wrote:

> In Global/OSX.gitignore
> <https://github.com/github/gitignore/pull/1583#discussion_r33900165>:
>
> > +
> > +# Icon must end with two \r
> > +Icon
> > +
> > +
> > +# Thumbnails
> > +._*
> > +
> > +# Files that might appear in the root of a volume
> > +.DocumentRevisions-V100
> > +.fseventsd
> > +.Spotlight-V100
> > +.TemporaryItems
> > +.Trashes
> > +.VolumeIcon.icns
> > +.temp
>
> Do you know what tool might be responsible for generating this .temp file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/github/gitignore/pull/1583/files#r33900165>.
>


-- 


  *Max Tuchman*
------------------------------------------

------------------------------------------
Contact me:
me@mbtuchman.com
mbtuchman@maxdev.us

*DISCLAIMER:*
The information contained in this e-mail and its attachments are intended
solely for the use of the individual(s) to whom it is addressed and
contains information that is confidential. If the reader of this e-mail is
not the intended recipient, you are hereby notified that any dissemination,
distribution, or copying of this e-mail is strictly prohibited. If you have
received this e-mail in error, please immediately notify the sender and
destroy the e-mail.
Thank You.

Comment (DigiCoon) [inline Global/OSX.gitignore]: That's the change.

**Comment by vgturtle127** [comment]
So just .temp files are being added to the .gitignore file?

**Comment by fdizile** [comment]
Correct.

**Comment by vgturtle127** [comment]
@fdizile Alright, I say we merge it.

:+1:

File: Global/OSX.gitignore [hunk 2]
```diff
@@ -1,24 +1,26 @@
-.DS_Store
-.AppleDouble
-.LSOverride
-
-# Icon must end with two \r
```
Comment (shiftkey) [inline Global/OSX.gitignore]: I can't merge this PR in it's current form due to the line endings change regressing this rule

**Comment by vgturtle127** [comment]
Can we just add the .temp? That should be simple, right? Just rebase and merge?

If we revert all the changes and then commit the single change it should all be fine.

**Comment by shiftkey** [comment]
Let's start this PR fresh - the line endings change means this is a no-go.

**Pull request closed without merging**